src/test/scala/uk/gov/nationalarchives/notifications/EcrScanIntegrationSpec.scala

```diff
@@ -0,0 +1,113 @@
+package uk.gov.nationalarchives.notifications
+import org.scalatest.prop.TableFor3
```

Comment:
Maybe add a line break between package and import statements, for consistency?

Reply:
I've fixed all of these, thanks. I'm not quite sure what happened because IntelliJ is set up to add a blank line between the package statement and the imports.